contracts/gov/user-keeper/GovUserKeeper.sol

```diff
@@ -513,10 +511,8 @@ contract GovUserKeeper is IGovUserKeeper, OwnableUpgradeable, ERC721HolderUpgrad
             uint256 totalPowerInTokens = nftInfo.totalPowerInTokens;
 
             for (uint256 i; i < nftIds.length; i++) {
-                nftsPower += totalPowerInTokens.ratio(
-                    snapshot.nftPower[nftIds[i]],
-                    totalNftsPower
-                );
+                (uint256 power, ) = nftContract.getNftPower(nftIds[i]);
```

Comment:
Please remove the collateral from the returned values

Reply:
done